content/s14_user_profiles/s14_p02_profile_editor.md

```diff
@@ -0,0 +1,101 @@
+Now that you've got your user profile page up, you'll need to let your users edit their information. In this lesson, you'll learn just how to do that. Then, the next challenge you must face is to make a similar form for an administrator. Stay tuned!
+
+### Edit Profile Form
+
+#### The Form Itself
+
+Your new `EditProfileForm` will be just a simple form for changing a users details, located in `main/forms.py`:
+
+```python
+class EditProfileForm(FlaskForm):
+    name = StringField("Name", validators=[Length(0, 64)])
+    location = StringField("Location", validators=[Length(0,64)])
+    bio = TextAreaField("Bio")
+    submit = SubmitField("Submit")
+```
+
+This has just a `TextAreaField` so that the user can go to town on their own life, since it's more than just a simple text box. Or say nothing at all, since the `Length` validator isn't specified!
```

Comment:
maybe reformulate "go to town on their own life", not sure this is totally understandable